com.basistech.m2e.code.quality.findbugs/META-INF/MANIFEST.MF

```diff
@@ -3,9 +3,9 @@ Bundle-ManifestVersion: 2
 Bundle-Name: M2Eclipse Project Configurator for Eclipse Findbugs
 Bundle-SymbolicName: com.basistech.m2e.code.quality.findbugs;singleton:=true
 Bundle-Version: 1.0.0.qualifier
-Require-Bundle: org.eclipse.m2e.jdt;bundle-version="[1.0.0,1.8.0)",
- org.eclipse.m2e.core;bundle-version="[1.0.0,1.8.0)",
- org.eclipse.m2e.maven.runtime;bundle-version="[1.0.0,1.8.0)",
+Require-Bundle: org.eclipse.m2e.jdt;bundle-version="[1.0.0,1.9.0)",
```

Comment:
What were the reason to put upper bound to 1.8.0?
I think the best is to put 2.0.0 upper bound as it is the semantic meaning a breaking API change, with 1.8.0, 1.9.0, 1.10.0 all api changes should ba backward compatible